<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>sobel</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="Line" name="line" >
    <property name="geometry" >
     <rect>
      <x>399</x>
      <y>10</y>
      <width>16</width>
      <height>536</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>408</x>
      <y>10</y>
      <width>382</width>
      <height>536</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>384</width>
      <height>536</height>
     </size>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="sizeConstraint" >
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Blur</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Gamma</string>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
       <property name="pageStep" >
        <number>25</number>
       </property>
       <property name="tracking" >
        <bool>false</bool>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="buddy" >
        <cstring>verticalSlider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maximum" >
        <number>999999</number>
       </property>
       <property name="value" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>51</x>
      <y>11</y>
      <width>341</width>
      <height>534</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>384</width>
      <height>536</height>
     </size>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>line</zorder>
   <zorder>label</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionWczytaj" />
    <addaction name="actionZapisz" />
   </widget>
   <widget class="QMenu" name="menuOp_2" >
    <property name="title" >
     <string>Działaj</string>
    </property>
    <widget class="QMenu" name="menuBinaryzacja" >
     <property name="title" >
      <string>Binaryzacja</string>
     </property>
     <addaction name="actionBinaryzacja_2" />
     <addaction name="actionLame" />
    </widget>
    <widget class="QMenu" name="menuBlur" >
     <property name="title" >
      <string>Blur</string>
     </property>
     <addaction name="actionAvg_blur" />
     <addaction name="actionGauss" />
    </widget>
    <addaction name="actionSzaro" />
    <addaction name="actionGradient" />
    <addaction name="menuBlur" />
    <addaction name="actionFiltr_medianowy" />
    <addaction name="actionDajesz" />
    <addaction name="actionCanny" />
    <addaction name="menuBinaryzacja" />
    <addaction name="actionHough" />
   </widget>
   <widget class="QMenu" name="menuPaszcza" >
    <property name="title" >
     <string>Paszcza</string>
    </property>
    <widget class="QMenu" name="menuCechy" >
     <property name="title" >
      <string>Cechy</string>
     </property>
     <addaction name="actionMaska_gradient_w" />
     <addaction name="actionOczy_houghem" />
    </widget>
    <addaction name="actionPrzytnij" />
    <addaction name="menuCechy" />
   </widget>
   <addaction name="menuOp" />
   <addaction name="menuOp_2" />
   <addaction name="menuPaszcza" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionGo" />
  </widget>
  <action name="actionWczytaj" >
   <property name="text" >
    <string>Wczytaj</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionZapisz" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Zapisz</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDajesz" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Sobel</string>
   </property>
   <property name="shortcut" >
    <string>S</string>
   </property>
  </action>
  <action name="actionBinaryzacja_2" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Otsu's</string>
   </property>
   <property name="shortcut" >
    <string>O</string>
   </property>
  </action>
  <action name="actionLame" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Lame</string>
   </property>
  </action>
  <action name="actionSzaro" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Szarość</string>
   </property>
   <property name="shortcut" >
    <string>Y</string>
   </property>
  </action>
  <action name="actionAvg_blur" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Avg blur</string>
   </property>
   <property name="shortcut" >
    <string>A</string>
   </property>
  </action>
  <action name="actionHough" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Hough</string>
   </property>
   <property name="shortcut" >
    <string>H</string>
   </property>
  </action>
  <action name="actionFiltr_medianowy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Filtr medianowy</string>
   </property>
   <property name="shortcut" >
    <string>M</string>
   </property>
  </action>
  <action name="actionGradient" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Gradient</string>
   </property>
   <property name="shortcut" >
    <string>G</string>
   </property>
  </action>
  <action name="actionGauss" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Gauss</string>
   </property>
  </action>
  <action name="actionCanny" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Canny</string>
   </property>
  </action>
  <action name="actionGo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Go!</string>
   </property>
  </action>
  <action name="actionPrzytnij" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Przytnij</string>
   </property>
  </action>
  <action name="actionMaska_gradient_w" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Maska gradientów</string>
   </property>
  </action>
  <action name="actionOczy_houghem" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Oczy houghem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
